container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/logs_aggregator_functions/implementations/vector/consts.go

```diff
@@ -21,17 +21,19 @@ const (
 	fileSinkIdSuffix = "file"
 	fileTypeId       = "\"file\""
 
-	// We store log files in the volume per-enclave, per-service
+	// We instruct vector to store log files per-year, per-week (00-53), per-enclave, per-service
 	// To construct the filepath, we utilize vectors template syntax that allows us to reference fields in log events
 	// https://vector.dev/docs/reference/configuration/template-syntax/
-	uuidLogsFilepath = "\"" + logsStorageDirpath + "{{ enclave_uuid }}/{{ service_uuid }}.json\""
+	baseLogsFilepath = "\"" + logsStorageDirpath + "%%Y/%%U/"
+
+	uuidLogsFilepath = baseLogsFilepath + "{{ enclave_uuid }}/{{ service_uuid }}.json\""
 
 	// Right now, we store duplicate logs files by service name and service's shortened uuid
 	// This is to enable retrieving logs by name and short uuid after enclaves have been stopped
 	// As right now, when an enclave is stopped all identifier info is lost
 	// TODO: Find a better way to capture or persist identifier info without storing duplicate log files
-	nameLogsFilepath      = "\"" + logsStorageDirpath + "{{ enclave_uuid }}/{{ service_name }}.json\""
-	shortUUIDLogsFilepath = "\"" + logsStorageDirpath + "{{ enclave_uuid }}/{{ service_short_uuid }}.json\""
+	nameLogsFilepath      = baseLogsFilepath + "{{ enclave_uuid }}/{{ service_name }}.json\""
+	shortUUIDLogsFilepath = baseLogsFilepath + "{{ enclave_uuid }}/{{ service_short_uuid }}.json\""
```

Comment:
out of curiosity, why do we use short uuids here and not the long ones? long ones have less chance of collisions, and I imagine there's not a big difference as to which one we're using here. Curious why we did not fallback on the safest option.